src/App.scss

```diff
@@ -28,7 +29,24 @@ $dars-panel-background-color: rgb(0 183 255);
 
 #dars {
   background-color: $dars-panel-background-color;
+}
+
+.hide {
+  width: 0;
+  -webkit-transition: width, 0.4s linear;
+  -moz-transition: width, 0.4s linear;
+  -ms-transition: width, 0.4s linear;
+  -o-transition: width, 0.4s linear;
+  transition: width, 0.4s linear;
+}
+
+.visible {
   width: 30%;
+  -webkit-transition: width, 0.4s linear;
+  -moz-transition: width, 0.4s linear;
+  -ms-transition: width, 0.4s linear;
+  -o-transition: width, 0.4s linear;
+  transition: width, 0.4s linear;
```

Comment:
I personally don't want to have a massive style in one file. I think it would be great to have separate style files for each component/file.

Reply:
or i can work on it on a separate pr